// Copyright 2023 Intrinsic Innovation LLC

syntax = "proto3";

package intrinsic_proto.icon;

// A 6x6 matrix.
message Matrix6d {
  repeated double data = 1 [packed = true];
}
